src/USER-OMP/pair_reaxc_omp.h

```diff
@@ -76,6 +76,7 @@ class PairReaxCOMP : public PairReaxC, public ThrOMP {
   }
 
  protected:
+  std::string fix_id;
```

Comment:
@ellio167 we are currently trying to avoid having STL (and other) headers included in style file headers. Would you mind making fix_id a `const char *` and use sprintf() or strcat() to create the unique string, as it is done in other places?

Reply:
sure, I'll be happy to do that.